app/src/main/java/uk/org/openseizuredetector/MainActivity.java

```diff
@@ -450,15 +451,15 @@ protected void onStart() {
         tv.setBackgroundColor(okColour);
         tv.setTextColor(okTextColour);
 
+        serverStatusRunnable.run();
+
         if (mUtil.isServerRunning()) {
             mUtil.writeToSysLogFile("MainActivity.onStart - Binding to Server");
             if (Objects.nonNull(mConnection)) {
                 if (!mConnection.mBound) mUtil.bindToServer(MainActivity.this, mConnection);
-                if (!mConnection.mSdServer.uiLiveData.isListeningInContext(this)){
```

Comment:
swapped with connectUiLiveDataRunner: , need to call and recall the same function until bound